cdc/model/owner.go

```diff
@@ -388,7 +303,6 @@ func (status *ChangeFeedStatus) Unmarshal(data []byte) error {
 
 // ProcInfoSnap holds most important replication information of a processor
 type ProcInfoSnap struct {
-	CfID      ChangeFeedID                  `json:"changefeed-id"`
-	CaptureID string                        `json:"capture-id"`
-	Tables    map[TableID]*TableReplicaInfo `json:"-"`
```

Comment:
It looks that `Tables` here is useless before? Have you ever test this?

Reply:
No usage in ticdc code base.